core/CMake_readelf.cmake

```diff
@@ -135,19 +135,24 @@ if (check_libc)
     OUTPUT_VARIABLE file_header_result
     )
 
-  # To determine the minimum version of glibc that we should support for packaging, we should
-  # check the Linux distributions that are known not to be rolling releases and offer long
-  # support, and then check the oldest option available that is not yet EOL. As of writing,
-  # these are:
-  #  * Debian Stretch, which is on glibc 2.24
-  #  * CentOS 7/RHEL 7, which is on glibc 2.17
-  #  * Ubuntu 16.04 LTS (Xenial), which is on glibc 2.23
+  # To determine the minimum version of glibc that we should support for packaging, we
+  # should check the Linux distributions that are known not to be rolling releases and
+  # offer long support, and then check the oldest option available that is not yet EOL.
+  # As June 2025, these are:
```

Comment:
Wondering why you mentioned "June 2025" instead of the current month. I see Ubuntu 20.04 is EOL 2025-04-15. RHEL was EOL June 2024, Debian Buster (just before Bullseye) was EOL-LTS 2024-06-30.

Reply:
I thought Ub20.04 EOL was 5/31/25: that's what I wrote at https://github.com/DynamoRIO/dynamorio/issues/5860#issuecomment-2706976008.  Searching again: I see both Apr 30 and May 31 in some results; the Ubuntu site says it ends "May".  Seems safest to stick with the current comment of June so it's clear 20.04 is over.

Reply:
You're right about 20.04: https://wiki.ubuntu.com/Releases. I can't find where I saw 2025-04-15 now.